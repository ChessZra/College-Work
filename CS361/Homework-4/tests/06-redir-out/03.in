file t3.txt
echo some data some data >t3.txt
md5sum t3.txt
md5sum t2t2t1.txt
rm t3.txt
