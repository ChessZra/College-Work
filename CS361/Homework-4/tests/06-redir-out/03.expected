cs361sh> t3.txt: cannot open `t3.txt' (No such file or directory)
cs361sh> cs361sh> bf688338c6a897c10cf05f8095afbfce  t3.txt
cs361sh> md5sum: t2t2t1.txt: No such file or directory
cs361sh> cs361sh> 
